site/src/components/checkbox.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Checkbox
+description: The checkbox component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D855%253A1014"
+---
+
+# Checkbox
+
+A selectable menu list from which an user can make one or more selections.
```

Comment:
I think that this may not quite be the right definition, perhaps something more akin to what MDN says? 
> A checkbox allows you to select single values for submission in a form

Reply:
Updated